stamp_service/search.py

```diff
@@ -8,16 +8,20 @@
 
 
 class S3Searcher:
-    def init(self, bucket_name, client=None):
+
+    buckets_dict = {}
```

Comment:
Why have this as a class attribute? Seems like just initializing in __init__ should be enough, unless I'm missing something

Reply:
was a place holder, in the init its enough